src/cascadia/TerminalSettingsEditor/Appearances.idl

```diff
@@ -67,8 +69,7 @@ namespace Microsoft.Terminal.Settings.Editor
         IInspectable CurrentAdjustIndistinguishableColors;
         Windows.Foundation.Collections.IObservableVector<Microsoft.Terminal.Settings.Editor.EnumEntry> AdjustIndistinguishableColorsList { get; };
 
-        Microsoft.Terminal.Settings.Model.ColorScheme CurrentColorScheme;
-        Windows.Foundation.Collections.IObservableVector<Microsoft.Terminal.Settings.Model.ColorScheme> ColorSchemeList { get; };
+        ColorSchemeViewModel CurrentColorScheme;
```

Comment:
This is still on the code-behind class; why isn't it in the viewmodel?

Reply:
Moved to the view model!